components/renku_data_services/notebooks/utils.py

```diff
@@ -99,18 +95,3 @@ def tolerations_from_resource_class(resource_class: crc_models.ResourceClass) ->
     for tol in resource_class.tolerations:
         output.append(Toleration(key=tol, operator="Exists"))
     return output
-
-
-async def get_user_secret(data_svc_url: str, user: AuthenticatedAPIUser) -> str | None:
-    """Get the user secret key from the secret service."""
-
-    async with httpx.AsyncClient(timeout=5) as client:
```

Comment:
For my education: is this result now coming from the `UserRepo` that has been added as a dependency into the `blueprints.py` above (now it is a library call, instead of a network one)?

Reply:
yes, now it calls the database repo directly. before it did an HTTP call to the respective endpoint

Reply:
this code was ported from renku-notebooks, where an http request made sense. but it doesn't make sense for a service to call itself

Reply:
I see, thank you! 🙏🏼